### PR TITLE
Add code

### PR DESCRIPTION
**Pull request by mizerawojciech**

### CODE REVIEW

File: Forte.EpiImageHotSpots/Forte.EpiImageHotSpots/Forte.EpiImageHotSpots.csproj [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net461</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="EPiServer.CMS.AspNet" Version="11.13.2" />
```
Comment (tomaszmadeyski) [inline Forte.EpiImageHotSpots/Forte.EpiImageHotSpots/Forte.EpiImageHotSpots.csproj]: I believe it is a good idea to use `allowedVersions` attribute here as you require specific version of a package.

File: Forte.EpiImageHotSpots/Forte.EpiImageHotSpots/ImageHotSpotsEditorDescriptor.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                .Last();
+
+            var lastVersion = this.contentLoader.Get<ImageHotSpotsBlockBase>(lastVersionReference.ContentLink);
+
+            
+            this.AddImageUrl(metadata, lastVersion);
+            this.AddBockPoints(metadata, lastVersion);
+        }
+
+        private void AddBockPoints(ExtendedMetadata metadata, ImageHotSpotsBlockBase hotSpotsBlock)
```
Comment (tomaszmadeyski) [inline Forte.EpiImageHotSpots/Forte.EpiImageHotSpots/ImageHotSpotsEditorDescriptor.cs]: missing `l` ?

File: Forte.EpiImageHotSpots/Forte.EpiImageHotSpots/ImageHotSpotsEditorDescriptor.cs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+using EPiServer.Cms.Shell.Extensions;
+using EPiServer.Core;
+using EPiServer.Shell.ObjectEditing;
+using EPiServer.Shell.ObjectEditing.EditorDescriptors;
+using EPiServer.Web.Routing;
+
+namespace Forte.EpiImageHotSpots
+{
+    [EditorDescriptorRegistration(TargetType = typeof(ImageHotSpots))]
+    [EditorDescriptorRegistration(TargetType = typeof(ImageHotSpots),
```
Comment (tomaszmadeyski) [inline Forte.EpiImageHotSpots/Forte.EpiImageHotSpots/ImageHotSpotsEditorDescriptor.cs]: do we need it twice?

File: Forte.EpiImageHotSpots/Forte.EpiImageHotSpots/Forte.EpiImageHotSpots.csproj (hunk 1 above)
Comment (mizerawojciech) [reply Forte.EpiImageHotSpots/Forte.EpiImageHotSpots/Forte.EpiImageHotSpots.csproj]: how can I check what versions can be allowed? Or should I just allow any version with arbitrary minimum

Comment (tomaszmadeyski) [reply Forte.EpiImageHotSpots/Forte.EpiImageHotSpots/Forte.EpiImageHotSpots.csproj]: I think allowing all versions which match major (11) is good enough

**Review by tomaszmadeyski — APPROVED** [review]

**Pull request merged**